src/Translumo/MVVM/ViewModels/LanguagesSettingsViewModel.cs

```diff
@@ -201,9 +203,22 @@ private async Task ReconfigureTts(Languages language, TTSEngines engine, Action
                     changeParameter,
                     _logger);
 
-                //if (TtsSettings.GetConfiguration<WindowsTtsConfiguration>().Enabled &&
-                if (engine == TTSEngines.WindowsTTS
-                && !TtsSettings.InstalledWinTtsLanguages.Contains(language))
+                if (!TtsSettings.IsLanguageSupportedInTtsEngine(engine, language))
```

Comment:
Also that checking can be encapsulated as first stage inside CreateSileroTtsCheckingStages or any other

Reply:
moved to CreateSileroTtsCheckingStages